lightmdb/views/contactus.py

```diff
@@ -2,7 +2,7 @@
 from lightmdb.forms import ContactForm
 from lightmdb.models import ContactMessage
 
-contactus = Blueprint('contactus', __name__)
+contactus = Blueprint('contact', __name__)
```

Comment:
contactus olarak yazdığım funsiyon ve sayfaların adını contact olarak değiştiğin için yaptım.